fv3core/stencils/remapping_part2.py

```diff
@@ -87,6 +87,10 @@ def compute(
     consv: float,
     do_adiabatic_init: bool,
 ):
+    saturation_adjustment = utils.cached_stencil_class(SatAdjust3d)(
+        cache_key="satadjust3d"
```

Comment:
perfect! could even put this inside of the if condition so it only gets compiled if the namelist says it is used. but this will get refactored into the init method of the parent object in a future pr anyways